main.nf

```diff
@@ -1,130 +1,575 @@
 #!/usr/bin/env nextflow
 /*
-vim: syntax=groovy
--*- mode: groovy;-*-
+========================================================================================
+                         nf-core/neutronstar
+========================================================================================
+ nf-core/neutronstar Analysis Pipeline.
+ #### Homepage / Documentation
+ https://github.com/nf-core/neutronstar
+----------------------------------------------------------------------------------------
 */
 
+def helpMessage() {
+    log.info nfcoreHeader()
+    log.info"""
 
-//##### Parameters
-version = 0.1
-def p = params
-p.genomesize = 0
-p.id = ''
-p.fastqs = ''
-p.sample = ''
-p.maxreads = 0
-p.bcfrac = 0
-p.outdir="."
-p.BUSCOdata = "$BUSCO_LINEAGE_SETS/eukaryota_odb9"
-def supernova_optional = {->
+    Usage:
+
+    The typical command for running the pipeline is as follows:
+
+
+    nextflow run -profile singularity nf-core/neutronstar --id assembly_id --fastqs fastq_path --genomesize 1000000
+
+    Mandatory arguments:
+      --id                          [Supernova parameter]
```

Comment:
It's fine to have `[Supernova parameter]` like this (I agree that it's clear), but could you add a description of what that parameter does after that?